examples/s3-import-mysql/main.tf

```diff
@@ -182,13 +174,13 @@ module "db" {
 
   # S3 import https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Procedural.Importing.html
   s3_import = {
-    source_engine_version = "8.0.27"
-    bucket_name           = aws_s3_bucket.import.id
+    source_engine_version = "8.0.28"
```

Comment:
Fixes this error:
<img width="1351" alt="Screen Shot 2022-07-24 at 9 56 45 PM" src="https://user-images.githubusercontent.com/69476188/180678665-877ea29a-9903-4521-a994-a4de90b3dc47.png">